Ldap/ad typo

MTRichards opened the pull request:
We had a typo in the ldap initial binding menu mouse over, so changed the text in all references to be correct.

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/owncloud/core/inspections/9f1dd621-7141-4c8a-9be5-e534fad32e6c) was created.

LukasReschke commented:
Can we squash the commits? 20 commits is a little bit much.

@DeepDiver1975 I'm not sure if the changes to the json files will persist as the CI job will overwrite them on the next sync to my understanding. Can you confirm? If so then we'd just need to change this at one place.

LukasReschke commented:
(Will squash this after @DeepDiver1975 commented)

MTRichards commented:
Where would I change it in one place that would pull it over? THAT would have been easier.

LukasReschke commented:
> Where would I change it in one place that would pull it over? THAT would have been easier.

We sync the translations with https://www.transifex.com/, so from my understanding what has to happen is:

1. We correct the English source string (no other one!)
2. Nightly sync with Transifex syncs the new string to the translators
3. Translators translate on https://www.transifex.com/
4. Changes are pulled back to ownCloud with nightly sync

At least that is what I think is happening here. Don't take my word for it though ;-) - So if this is the case we would just need to change the source string. The other changes would get replaced with step 4 anyways.

An exception are stable branches were no translations are synced. The changes would not get overwritten on those.

MTRichards commented:
Thanks!

MorrisJobke commented on apps/user_ldap/l10n/cs_CZ.json:
```diff
@@ -76,7 +76,7 @@
     "1. Server" : "1. Server",
     "%s. Server:" : "%s. Server:",
-    "Adds a new and blank configuration" : "Přidá novou prázdnou konfiguraci",
+    "Add a new and blank configuration" : "Přidá novou prázdnou konfiguraci",
```
These files are generated by a script.

MorrisJobke commented on apps/user_ldap/templates/part.wizard-server.php:
```diff
@@ -25,7 +25,7 @@
 		<button type="button" id="ldap_action_add_configuration"
 			name="ldap_action_add_configuration" class="icon-add"
-			title="<?php p($l->t('Adds a new and blank configuration'));?>">&nbsp;</button>
+			title="<?php p($l->t('Add a new and blank configuration'));?>">&nbsp;</button>
```
This change is fully sufficient :P

MorrisJobke commented:
@LukasReschke To answer your question: Yes it will be pulled and yes please remove all but the php file change ;)

MorrisJobke commented:
@MTRichards Thanks for your work

LukasReschke committed:
Dropped an S from the mouseover

We had a typo in the ldap initial binding menu mouse over, so changed the text in all references to be correct.

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/owncloud/core/inspections/2feeb4ce-22a2-4aee-9abf-e44bf6dd0ea9) was created.

LukasReschke commented:
PR adjusted.

LukasReschke commented:
:+1:

Xenopathic commented:
:+1:
